uaa/src/test/java/org/cloudfoundry/identity/uaa/login/TokenEndpointDocs.java

```diff
@@ -213,6 +220,8 @@ void getTokenUsingAuthCodeGrant() throws Exception {
                 parameterWithName("code").description(codeDescription).attributes(SnippetUtils.constraints.value("Required"), SnippetUtils.type.value(STRING)),
                 grantTypeParameter.description("the type of authentication being used to obtain the token, in this case `authorization_code`"),
                 clientSecretParameter.description("<small><mark>UAA 75.21.0</mark></small> Optional and can be omitted if client has configured allowpublic and [PKCE](https://tools.ietf.org/html/rfc7636) with `code_challange_method=S256` is used to create to `code`."),
```

Comment:
It's not part of this PR, but shouldn't this be `code_challenge_method`?

Reply:
Thanks for finding it, sure was wrong in past